### PR TITLE
Adds separate credentials file

### PR DESCRIPTION
**Pull request by matt-bernhardt**
Signed-off-by: Matt Bernhardt <mjbernha@mit.edu>

### CODE REVIEW

**Comment by matt-bernhardt** [comment]
Tagging @JPrevost in case you aren't already

File: includes/include_mongo_connect.php [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+
+	require_once($_SERVER['DOCUMENT_ROOT'] . "/includes/credentials.php");
+
 	try {
 		// connect to Mongo
 		switch ($_SERVER["SERVER_NAME"]) {
 			case "oastats-dev.mit.edu":
-				$m = new Mongo('mongodb://libdb-dev.mit.edu:27017');
+				// $m = new Mongo( $mongo["dev"] );
+				$m = new Mongo( $mongo["prod"] );
```
Comment (JPrevost) [inline includes/include_mongo_connect.php]: Is your intent to use one value for all of these cases now and just change it to what you want in the included file? If so, you can remove all of these cases and just set it once. If not, you need to update lines 10 and 17 to use different values than line 23.

**Comment by matt-bernhardt** [comment]
In the past we've had the dev and test applications pointing at a dev database server that had faked data. For the moment, we've got them all set to pull live data to make tuning the visualizations more realistic; would you recommend changing this strategy and just get better faked data in the dev db server?

I've used the flexibility this approach offers in the past, being able to point different environments at different data sources - even if currently that isn't the case.

**Comment by JPrevost** [comment]
If this does what you want, then :+1: from me.

My personal thoughts on how to handle what you want to do (if I understand it) would be to have a single `$mongo["server_url_or_whatever"]` in your `credentials.php` file and then just change that value to whatever you want whenever you want. That allows for removing all of the cases from the code while still maintaining the same amount of flexibility (the flexibility is now in the config file instead of the code).

I assume all the mongo stuff is read only for this app so point at prod seems fine from dev/test/local in general too.

**Pull request merged**